lib/index.js

```diff
@@ -1,7 +1,6 @@
-'use strict'
+import {Transform} from 'node:stream'
```

Comment:
Although I agree that using `node:` is correct, I use this in a bunch of places with WebPack replacing `stream` with `browserify-stream`.  This will require some effort to get working, and break those project's downstream even more.  Let's just use `stream` for now, please.